gadget/gadget-amd64.yaml

```diff
@@ -25,12 +25,22 @@ volumes:
         type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
         size: 1200M
         update:
-          edition: 2
+          edition: 3
         content:
           - source: grubx64.efi
-            target: EFI/boot/grubx64.efi
+            target: EFI/BOOT/grubx64.efi
```

Comment:
This is right.

I was a bit confused on why we still need this. But shim is configured to boot `\bootx64.efi` (`DEFAULT_LOADER`). And `generate_path_from_image_path` from shim considers paths that do not start with `\EFI\` to be relative.

I wonder if it would be better to sort these content entries bey target directory. So it is easier to see what goes  in `EFI\BOOT` and what goes in `EFI\ubuntu`.

Reply:
Thanks, good to know. I totally agree about sorting, I'll fix that in a moment.

Reply:
@alfonsosanchezbeato do you think `grub${arch}.efi` is required in `EFI/BOOT/` because of this?

Reply:
You are right. This is how it seems documented.